README.md

```diff
@@ -1,18 +1,55 @@
-# apollo-cli
+# rover-cli
 > ✨🤖 🐶 the new CLI for apollo
 
 [![Tests](https://github.com/apollographql/apollo-cli/workflows/Tests/badge.svg)](https://github.com/apollographql/apollo-cli/actions?query=workflow%3ATests)
 ![Stability: Experimental](https://img.shields.io/badge/stability-experimental-red)
 [![Netlify Status](https://api.netlify.com/api/v1/badges/1646a37a-eb2b-48e8-b6c9-cd074f02bb50/deploy-status)](https://app.netlify.com/sites/apollo-cli-docs/deploys)
 
-This is the home of the new CLI for Apollo's suite of GraphQL developer productivity tools.
+This is the home of `rover`, the new CLI for Apollo's suite of GraphQL developer productivity tools.
 
-This repo is organized as a [`cargo` workspace], containing two related projects:
+## Command-line options
+
+```
+✨🤖🐶 the new CLI for Apollo
+
+USAGE:
+    rover [OPTIONS] <SUBCOMMAND>
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -l, --log <log-level>     [default: debug]  [possible values: error, warn, info,
+                             debug, trace]
+
+SUBCOMMANDS:
+    config     ⚙️  Rover configuration
+    help       Prints this message or the help of the given subcommand(s)
+    partial    *️⃣  Federated schema/graph commands
+    schema     ⏺  Non-federated schema/graph commands
```

Comment:
i think this is outdated now since we merged #103
```suggestion
    subgraph   *️⃣  Federated schema/graph commands
    graph      ⏺  Non-federated schema/graph commands
```